src/integ_test_resources/android/amplify/integration/cdk/schemas/ScenarioA.graphql

```diff
@@ -0,0 +1,35 @@
+type ScenarioATest1Post @model @auth(rules: [{ allow: owner }]) {
```

Comment:
Unless we document them somewhere public, "A" and "1" have no meaning. Can we rename these models to self-document? Or alternately, can we add a `README.md` that explains them?